modules/ocf/manifests/ssl/lets_encrypt/dns_common.pp

```diff
@@ -8,7 +8,9 @@
 
   # we use this package in the le_cert_info fact, to gather information about
   # whatever certs are currently present
-  package { 'python3-cryptography':; }
+  ocf::repackage { 'python3-cryptography':
+    backport_on => ['stretch'],
+  }
```

Comment:
Take a gander at https://github.com/ocf/puppet/pull/449#discussion_r247018609 if you haven't. If we are directly using this for things besides letsencrypt, we should probably just move it to modules/ocf/manifests/packages.pp or something.

Reply:
I don't think this is as necessary for this repo, since it's not running as a fact. It only gets run by users, so we don't have to worry about it running during the bootstrap process.

Reply:
However since it's depended upon in more than one place, moving it to `packages.pp` is the right move, I'll do that